packages/core/test/coreConfiguration.ts

```diff
@@ -72,6 +74,7 @@ export type MapInitConfigurationKey<Key extends string> =
           | 'internalAnalyticsSubdomain'
           | 'replica'
           | 'enableExperimentalFeatures'
+          | 'usePciIntake'
```

Comment:
**💬 suggestion:** ‏include this parameter in the Telemetry Configuration as it's not sensitive